public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -114,6 +117,13 @@ export class StatPanel extends PureComponent<PanelProps<PanelOptions>> {
       sparkline: options.graphMode !== BigValueGraphMode.None,
       timeZone,
     });
+
+    // If there are custom prefixes or suffixes to format values, format the values
+    if (customConfig?.prefix || customConfig?.suffix) {
```

Comment:
Pretty sure we can remove the null check since above we did `?? {}`
```suggestion
    if (customConfig.prefix || customConfig.suffix) {
```

Reply:
agreed